packages/core/src/RenderingEngine/Viewport.ts

```diff
@@ -147,12 +152,16 @@ class Viewport implements IViewport {
    * @param options - The viewport options to set.
    * @param immediate - If `true`, renders the viewport after the options are set.
    */
-  public setOptions(options: ViewportInputOptions, immediate = false): void {
+  public setOptions(
+    options: ViewportInputOptions,
+    immediate = false,
+    storeAsInitialCamera = false
```

Comment:
can we avoid having `storeAsInitialCamera` here?

Reply:
Yes, this could move into the options.DisplayArea